queries/typst/injections.scm

```diff
@@ -1,2 +1,7 @@
 ((comment) @injection.content
   (#set! injection.language "comment"))
+
+(raw_blck
+  (ident) @_lang
+  (blob) @injection.content
+  (#set-lang-from-info-string! @_lang))
```

Comment:
This is deprecated due to upstream changes.

Reply:
Oh sorry, i just looked at what markdown query did.

Reply:
i checked some other `injections.scm`  and dont see a way *other than manually writing it for every language*
 
 Am i missing something?

Reply:
`@injection.language`

Reply:
Yes, yes you do
https://github.com/neovim/neovim/blob/bd5008de07d29a6457ddc7fe13f9f85c9c4619d2/runtime/lua/vim/treesitter/languagetree.lua#L759

Reply:
Also you can always set it through `vim.treesitter.language.register, which we do set in `parsers.lua`